jetty-core/jetty-client/src/main/java/org/eclipse/jetty/client/AbstractConnectionPool.java

```diff
@@ -121,24 +113,25 @@ public void setMaxDuration(long maxDurationInMs)
         this.maxDurationNanos = TimeUnit.MILLISECONDS.toNanos(maxDurationInMs);
     }
 
-    protected int getMaxMultiplex()
+    protected int getInitialMaxMultiplex()
     {
-        return pool.getMaxMultiplex();
+        return initialMaxMultiplex;
     }
 
-    protected void setMaxMultiplex(int maxMultiplex)
+    protected void setInitialMaxMultiplex(int initialMaxMultiplex)
     {
-        pool.setMaxMultiplex(maxMultiplex);
+        this.initialMaxMultiplex = initialMaxMultiplex;
     }
 
-    protected int getMaxUsageCount()
+    @ManagedAttribute(value = "The maximum amount of times a connection is used before it gets closed")
```

Comment:
The fact that it defaults to 0 and that `0-` values disable the mechanism should be documented.